tools/flaky-test-reporter/config.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// config.go contains configurations for flaky tests reporting
+
+package main
+
+import (
+	"github.com/knative/test-infra/shared/prow"
+)
+
+const (
+	buildsCount         = 10 // Builds to be analyzed
```

Comment:
I believe these are arbitrary values. If so, better state this here, so we're confident about experimenting with them.

Reply:
done